hs_core/views/utils.py

```diff
@@ -698,6 +781,26 @@ def unzip_file(user, res_id, zip_with_rel_path, bool_remove_original):
     hydroshare.utils.resource_modified(resource, user, overwrite_bag=False)
 
 
+def _get_destination_filename(file, unzipped_foldername):
+    split = file.split("/" + unzipped_foldername + "/", 1)
+    destination_file = os.path.join(split[0], split[1])
+    return destination_file
```

Comment:
@sblack-usu It'd be good to add some comment into this internal function since I am confused on what it does.

Reply:
Good point, I added some documentation.